packages/language-server/test/initialize.test.js

```diff
@@ -22,6 +22,13 @@ test('initialize', async () => {
     {typescript: {enabled: true, tsdk}}
   )
   assert.deepEqual(initializeResponse, {
+    autoInsertion: {
+      configurationSections: [
+        'javascript.autoClosingTags',
+        'typescript.autoClosingTags'
+      ],
+      triggerCharacters: ['>', '>']
```

Comment:
It repeats because `>` requires matching `javascript.autoClosingTag` and `typescript.autoClosingTags` at the same time. But maybe I should change the format to:
```js
    autoInsertion: {
      configurationSections: [
       ['javascript.autoClosingTags', 'typescript.autoClosingTags']
      ],
      triggerCharacters: ['>']
	}
```